testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1788,6 +1788,63 @@ QUnit.module('selection', moduleSetup, () => {
     });
 });
 
+QUnit.module('selectByClick option', ()=> {
+    QUnit.test('selectByClick option should be changed during initialization', function(assert) {
+        const list = $('#list').dxList({ items: [1, 2], selectByClick: false }).dxList('instance');
+        const isSelectByClick = list.option('selectByClick');
+
+        assert.strictEqual(isSelectByClick, false);
+    });
+
+    QUnit.test('selectByClick option should be changed after initialization', function(assert) {
+        const list = $('#list').dxList({ items: [1, 2] }).dxList('instance');
+
+        list.option('selectByClick', false);
+
+        const isSelectByClick = list.option('selectByClick');
+
+        assert.strictEqual(isSelectByClick, false);
+    });
```

Comment:
I don't think these 2 tests are useful. If we do `dxList({ selectByClick: false })` we pass `false` to an option. So it's clear that option value is passed) 
If's like write a test `let a = 2; assert(a === 2)`

Reply:
It is not clear to me personally that the value will be transmitted exactly.
This may be due to the fact that I do not fully understand the operation of the component

Reply:
Or the point is that I check it in the following tests?

Reply:
We may remove this test and add tests with test scenario the same that follow tests have